airtime_mvc/public/js/airtime/library/library.js

```diff
@@ -1277,7 +1277,7 @@ var AIRTIME = (function(AIRTIME) {
                     }
                     // remove 'Add to smart block' option if the current
                     // block is dynamic
-                    if ($('input:radio[name=sp_type]:checked').val() === "1") {
+                    if ($('input:radio[name=sp_type]:checked').val() === "0") {
```

Comment:
i can't see what this actually does... even before this change i see no change in the enable/disabling of the + button above, nor to the contextual menus for tracks. i think this feature was broken, but honestly not a priority for me - people can just use conventional playlists.

![screenshot from 2018-12-12 04-33-13](https://user-images.githubusercontent.com/36803137/49864056-37b92580-fdc7-11e8-9392-024d36f5906a.png)

Reply:
If you are editing a smartblock and you navigate the left column to tracks a button at the top says "+ add to smartblock" this isn't grayed out when you have a dynamic smartblock but gives an error message "You cannot add tracks to dynamic blocks." it works for static smartblocks. So I don't think it's a big deal.